fixieai/agents-ts/src/__tests__/serve.ts

```diff
@@ -6,7 +6,7 @@ import tempy from 'tempy';
 import type { PromiseType } from 'utility-types';
 import serve from '../serve';
 
-const agentPackagePath = path.resolve(__dirname, '..', 'fixtures');
+const agentPackagePath = path.resolve(__dirname, '..', 'fixtures', 'normal');
```

Comment:
I split the test fixtures into `normal` and `watch`, because the `watch` mode tests need to be more constrained, but I didn't want those constraints to apply to the rest of the test suite.

Some of this diff is just noise related to that, but isn't actually a meaningful change.